acceptance/tests/i18n/modules/puppet_apply.rb

```diff
@@ -46,7 +46,7 @@
     step "Run puppet apply of a module with language #{agent_language} and verify the translations" do
       step 'verify custom fact translations' do
         on(agent, puppet("apply -e \"class { 'i18ndemo': filename => '#{type_path}' }\"", 'ENV' => shell_env_language)) do |apply_result|
-          assert_match(/Error:.*\w+-i18ndemo fact: これは\w+-i18ndemoからのカスタムファクトからのレイズです/, apply_result.stderr, 'missing translation for raise from ruby fact')
+          assert_match(/Warning:.*\w+-i18ndemo fact: これは\w+-i18ndemoからのカスタムファクトからのレイズです/, apply_result.stderr, 'missing translation for raise from ruby fact')
```

Comment:
@joshcooper can you think of any reason these would be reporting a warning rather than an error when testing in a git-based system?

Reply:
I'm guessing because git will install ruby facter via bundler? But ruby facter doesn't know anything about translations...

Reply:
it's still getting the translation correct though, just not the type of result it gets.....

Reply:
is it facter or puppet that's throwing these errors/warnings?

Reply:
@eputnam halp? We're thinking it's native facter vs ruby facter, but before I start digging into the module itself, I was wondering if you had any insights?

Reply:
No insights here, sorry :\